app/UpdateStore.tsx

```diff
@@ -7,24 +7,22 @@ import { PageHeader } from "./ui/PageHeader";
 import { FormLabel } from "./ui/FormLabel";
 import { FormGroup } from "./ui/FormGroup";
 import { FormControl } from "./ui/FormControl";
+import { Store } from "./types";
 
 export function UpdateStore(props: {
   onUpdateStoreSuccess: () => void;
   onBackClick: () => void;
-  //   prefilledDisplayName: string;
-  //   prefilledFullName: string;
-  //   prefilledSuburb: string;
+  storeDetails?: Store;
 }): ReactElement {
   const [errorMessage, setErrorMessage] = useState("");
+  const { onUpdateStoreSuccess, onBackClick, storeDetails } = props;
 
-  const [displayName, setdisplayName] = useState("");
-  const [fullName, setFullName] = useState("");
-  const [suburb, setSuburb] = useState("");
-
-  const { onUpdateStoreSuccess: onUpdateStoreClick, onBackClick } = props;
+  const [displayName, setdisplayName] = useState(storeDetails?.displayName);
+  const [fullName, setFullName] = useState(storeDetails?.fullName);
+  const [suburb, setSuburb] = useState(storeDetails?.suburb);
   return (
     <Wrapper>
-      <PageHeader> STORE NAME</PageHeader>
+      <PageHeader>{storeDetails?.displayName.toUpperCase()}</PageHeader>
```

Comment:
I don't think this needs to be upper cased, it just so happens in the mocks that `TANG`'s name is all upper

Reply:
okay well I assumed you made it that way on purpose so I copied it 
can you make notes on what actually matters in the mocks then cos then it feels like I'm just arbitrarily deciding what's important or not

Reply:
Not really possible for designers to cover all cases or predict how the engineer will interpret the mocks, eg in this case I thought it was sensible that the name up the top was just exactly what the display name is - you can also see that the full name has `TANG` in caps but the rest of the name isn't upper-cased. 

In general we would just check with the designers if anything seems off or ambiguous, eg
<img width="901" alt="Screenshot 2023-11-19 at 1 43 42 pm" src="https://github.com/auberginearms/groceryDatabaseApp/assets/53922879/307fba0f-451c-44c0-a1a5-a384356bf881">

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/20/commits/b2950c317506872d07d5c2d8f5ddc5048f6d3b89